### ISSUE TITLE
Some symbols are not supported under Windows file system

### ISSUE THREAD

**Issue by Sicmatr1x** [hellojukay/dl-talebook#5]
Some symbols are not supported under Windows, such as '?'

For example, if you download the book title is this:
```
Wha's Like Us? (Say It in Scots!).epub
```

will got an error:

```
open G:\EBook\taleServer\Wha's Like Us? (Say It in Scots!).epub: The filename, directory name, or volume label 
syntax is incorrect.
```

**Comment by syhily** [comment]
It's quite easy to fix. The naming conversion in windows is here: https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#naming-conventions

**Comment by syhily** [comment]
@Sicmatr1x I have fixed this issue on my personal fork. You can download the distribution on the [release page](https://github.com/syhily/dl-talebook/releases).

**Comment by hellojukay** [comment]
@Sicmatr1x  this issue is fixed in v0.4.0 ,  can you try this version？
